ui/tests/acceptance/secure-variables-test.js

```diff
@@ -103,9 +103,8 @@ module('Acceptance | secure variables', function (hooks) {
     await percySnapshot(assert);
 
     await click(fooLink);
-    assert.equal(
-      currentURL(),
-      '/variables/var/a/b/c/foo0',
+    assert.ok(
+      currentURL().includes('/variables/var/a/b/c/foo0'),
```

Comment:
`blocking`: The assertion should include that the `namespace` is included in the URL otherwise the page will `404` or return the wrong variable.

Reply:
I think this test is still failing.

Reply:
Hey! Noticed this was marked as resolved, but I didn't see the assertion change.